packages/apollo-angular/src/index.ts

```diff
@@ -5,5 +5,75 @@ export {Mutation} from './Mutation';
 export {Subscription} from './Subscription';
 export {SelectPipe} from './SelectPipe';
 export {ApolloModule} from './ApolloModule';
-export {APOLLO_OPTIONS, APOLLO_NAMED_OPTIONS} from './tokens';
-export {SubscriptionResult} from './types';
+export {APOLLO_OPTIONS, APOLLO_NAMED_OPTIONS, APOLLO_FLAGS} from './tokens';
+export {
+  SubscriptionResult,
+  Flags,
+  WatchQueryOptions,
+  ExtraSubscriptionOptions,
+} from './types';
+export {HttpLinkModule} from './http/HttpLinkModule';
+export {HttpLink, HttpLinkHandler} from './http/HttpLink';
+export {Options as HttpLinkOptions} from './http/types';
+export {HttpBatchLink, HttpBatchLinkHandler} from './http-batch/HttpBatchLink';
+export {HttpBatchLinkModule} from './http-batch/HttpBatchLinkModule';
+export {BatchOptions as HttpBatchLinkOptions} from './http-batch/types';
+export {
+  Observable as LinkObservable,
+  // ApolloClient
+  ApolloClient,
+  ApolloClientOptions,
+  DefaultOptions,
+  // core/ObservableQuery
+  ObservableQuery,
+  FetchMoreOptions,
+  UpdateQueryOptions,
+  ApolloCurrentQueryResult,
+  // core/watchQueryOptions
+  QueryOptions,
+  MutationOptions,
+  SubscriptionOptions,
+  FetchPolicy,
+  WatchQueryFetchPolicy,
+  ErrorPolicy,
+  FetchMoreQueryOptions,
+  SubscribeToMoreOptions,
+  MutationUpdaterFn,
+  // core/networkStatus
+  NetworkStatus,
+  // core/types
+  ApolloQueryResult,
+  Resolvers,
+  // core/LocalState
+  Resolver,
+  LocalStateFragmentMatcher,
+  // errors
+  isApolloError,
+  ApolloError,
+  // cache
+  Cache,
+  DataProxy,
+  ApolloCache,
+  InMemoryCache,
+  InMemoryCacheConfig,
+  defaultDataIdFromObject,
+  // links
+  GraphQLRequest,
+  Operation,
+  FetchResult,
+  NextLink,
+  RequestHandler,
+  empty,
+  from,
+  split,
+  concat,
+  execute,
+  ApolloLink,
+  fromError,
+  ServerError,
+  throwServerError,
+  // utils
+  getMainDefinition,
```

Comment:
`getMainDefinition` is not exported by `@apollo/client/core` (version 3.0.0-beta.30) but `@apollo/client/utilities`. Is it a apollo-angular "bug" or related to apollo client?

Reply:
It was exported but now it's not... Oh Apollo

Can't do: 

```ts
export { HttpLink } from './http-link';
export * from '@apollo/client/core'
```

Because the compiled output does something like that:

```js
exports.HttpLink = httpLink_1.HttpLink

Object.keys(apollo_client_core_1).forEach(key => {
  if (key !== 'default') exports[key] = apollo_client_core_1[key];
});
```

Then HttpLink is different from what it should be. I'm not quite sure how to solve it other than export all or most used identifiers from `@apollo/client/core`.

The main goal is to make Angular folks avoid using `@apollo/client` because it contains React.

Reply:
@jonathanheilmann `2.0.0-beta.2`